boto/connection.py

```diff
@@ -1105,8 +1105,6 @@ def __init__(self, aws_access_key_id=None, aws_secret_access_key=None,
     def _required_auth_capability(self):
         return []
 
-    def get_utf8_value(self, value):
```

Comment:
Also add this back in.  Other parts of the codebase relied on this method:

```
<...>/boto/boto $ grep "get_utf8_value(" . -r
./mturk/connection.py:            value = connection.get_utf8_value(value)
./mturk/connection.py:            value = connection.get_utf8_value(value)
[...]
```

Reply:
Ah, good catch! Will re-add.